onnxruntime/core/framework/sequential_execution_plan.h

```
@@ -33,7 +33,7 @@ struct AllocPlanPerValue {
   bool create_fence_if_async{false};
 
  public:
-  AllocPlanPerValue() : location(CPU, OrtArenaAllocator) {}
+  AllocPlanPerValue() : location(CPU, Invalid) {}
 };
```

Default to Invalid so we know if it hasn't been set to a meaningful value.